app/routes.py

```diff
@@ -1,2 +1,293 @@
 from flask import Blueprint
+from app import db
+from app.models.task import Task
+from app.models.goal import Goal
+from flask import request, Blueprint, make_response, jsonify
+from datetime import datetime
+from app.slack import slack_message
+import os
+import requests
 
+
+tasks_bp = Blueprint("tasks", __name__, url_prefix="/tasks")
+goals_bp = Blueprint("goals", __name__, url_prefix="/goals")
+
+
+# -------------- CRUD for Tasks ---------------------------
+
+
+@tasks_bp.route("", methods=["GET"])
+def handle_tasks_get():
+    """
+    - Get all saved tasks.
+    - Get all saved tasks filtered by title in asc and desc order.
+    """
+
+    if not request.args:
+        tasks = Task.query.all()
+    else:
+        sort_query = request.args.get("sort", default=None, type=str)
+
+        if sort_query == "desc":
+            tasks = Task.query.order_by(Task.title.desc())
+        elif sort_query == "asc":
+            tasks = Task.query.order_by(Task.title.asc())
+        else:
+            return jsonify(
+                "Parameter sort is not defined. Use ?sort=desc or ?sort=asc to sort the tasks by title."), 404
+
+    tasks_response = [task.to_json() for task in tasks]
+
+    return jsonify(tasks_response)
+
+
+@tasks_bp.route("/<task_id>", methods=["GET"])
+def handle_one_task_get(task_id):
+    """
+    Get specific tasks by id.
+    """
+    task = Task.query.get(task_id)
+
+    if task is None:
+        return jsonify(None), 404
```

Comment:
We could do this in one line, like so:

```suggestion
    task = Task.query.get_or_404(task_id)
```